trompace/queries/templates.py

```diff
@@ -0,0 +1,97 @@
+# Templates for generate GraphQL queries for mutations.
+
+# To be added EntryPoint, ControlAction, PropertyValueSpecification and Property
+from trompace.exceptions import UnsupportedLanguageException
+from . import make_parameters, MUTATION
+from ..constants import SUPPORTED_LANGUAGES
+
+
+
+
+def query_create(identifier: str, query_string: str, name=None, publisher=None, contributor=None, creator=None,
+                    source=None, subject=None, description=None, language=None, coverage=None, date=None,
+                    disambiguatingDescription=None, relation=None, _type=None, _searchScore=None, additionalType=None,
+                    alternateName=None, image=None, sameAs=None, url=None, additionalName=None,
+                    award=None, birthDate=None, deathDate=None, familyName=None, gender=None, givenName=None,
+                    honorificPrefix=None, honorificSuffix=None, jobTitle=None, knowsLanguage=None):
+    """Returns a mutation for updating an object.
```

Comment:
you've copied this from a different file without updating the documentation. Please be careful doing this. There's no point in putting time into documentation if it's not correct.